src/wallet/components/Wallet.tsx

```diff
@@ -1,19 +1,36 @@
 'use client';
 
+import { Avatar, Name } from '@/identity';
 import { Draggable } from '@/internal/components/Draggable/Draggable';
 import { useIsMounted } from '@/internal/hooks/useIsMounted';
 import { useOutsideClick } from '@/internal/hooks/useOutsideClick';
 import { useTheme } from '@/internal/hooks/useTheme';
 import { findComponent } from '@/internal/utils/findComponent';
 import { cn } from '@/styles/theme';
-import { Children, useMemo, useRef } from 'react';
-import type { WalletReact, WalletSubComponentReact } from '../types';
+import {
+  Children,
+  type ReactNode,
+  isValidElement,
+  useMemo,
+  useRef,
+} from 'react';
+import type { WalletReact } from '../types';
 import { getWalletDraggableProps } from '../utils/getWalletDraggableProps';
 import { ConnectWallet } from './ConnectWallet';
 import { WalletAdvanced } from './WalletAdvanced';
 import { WalletDropdown } from './WalletDropdown';
 import { WalletProvider, useWalletContext } from './WalletProvider';
 
+const defaultWalletChildren = (
+  <>
+    <ConnectWallet>
+      <Avatar className="h-6 w-6" key="avatar" />
+      <Name key="name" />
+    </ConnectWallet>
+    <WalletDropdown />
+  </>
+);
+
 export const Wallet = ({
```

Comment:
nit: since we're refactoring, i think we should make this a proper function not an arrow function